cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -71,9 +95,9 @@
         <h3 class="o-post-preview_title">
             {{ settings.heading | safe }}
         </h3>
-        {% if settings.event_location %}
+        {% if settings.event_start_dt %}
             <span class="o-post-preview_subtitle h6">
-            {{settings.event_location}} - {{ time.render(settings.event_date) }}
+            {{ settings.event_city }}, {{ settings.state }} {{ settings.event_venue if settings.event_venue else '' }} - {{ time.render(settings.event_start_dt) }}
```

Comment:
I don't know, but it's hard to review due to the horizontal scrolling so it'd be nice to break it up a bit